CMakeLists.txt

```diff
@@ -34,7 +34,24 @@ add_definitions(-DNOMINATIM_VERSION="${NOMINATIM_VERSION}")
 
 set(BUILD_TESTS off CACHE BOOL "Build test suite" FORCE)
 set(WITH_LUA off CACHE BOOL "Build with lua support" FORCE)
-add_subdirectory(osm2pgsql)
+set(WWW_PUBLIC /var/www/nominatim CACHE PATH "Public www root")
+set(WWW_APP ${CMAKE_INSTALL_PREFIX}/share/nominatim CACHE PATH "Web application root")
+find_program(OSM2PGSQL osm2pgsql DOC "System installed osm2pgsql")
```

Comment:
Gotcha.
And yes, a different name would work. The date seemed to coincide with latest release, but now that I'm looking at hashes, they are indeed different.